hyperid.js

```diff
@@ -3,7 +3,7 @@
 const uuidv4 = require('./uuid')
 const parser = require('uuid-parse')
 const maxInt = Math.pow(2, 31) - 1
-const Buffer = require('buffer').Buffer
+const Buffer = require('buffer/').Buffer
```

Comment:
```suggestion
const Buffer = require('buffer/').Buffer
```

Please use the global `Buffer` object if it is available

Reply:
```javascript
function loadBuffer () {
  const b = require('buffer')
  // use third party module if no Buffer module
  return b
    ? b.Buffer
    : require('buffer/').Buffer
}
```
I have added a function to load Buffer, please review my new commit if have time.